Assignment1/src/main/java/Classes/CalculateTax.java

```diff
@@ -0,0 +1,91 @@
+package Classes;
+
+import java.util.*;
+import java.lang.*;
+
+public class CalculateTax {
+
+    static double totalTax = 0.0;
+    static int totalItems = 0;
```

Comment:
use proper access modifier. Read [this](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html)